doc/tools/pkcs11-tool.1.xml

```diff
@@ -646,6 +664,15 @@
 					</para></listitem>
 				</varlistentry>
 
+				<varlistentry>
+					<term>
+						<option>--iv</option> <replaceable>data</replaceable>
+					</term>
+					<listitem><para>Initialization vector for symmetric ciphers.
+					<replaceable>data</replaceable> - hexadecimal numbers i.e. "000013aa7bffa0".
```

Comment:
I would suggest that the description of the options in the manual pages should be in the whole sentences, such as "The `data` is hexadecimal number, i.e. ..."